src/styles.css

```diff
@@ -0,0 +1,1198 @@
+}
+
+.contact-form button {
+  height: 48px;
+  width: 148px;
+  margin-left: -15%;
+}
+
+.modal-example {
+  display: none;
+  flex-direction: column;
+  background: #fff;
+  border-radius: 20px;
+  margin: 5%;
+  width: 80%;
+  padding: 10px;
+  position: fixed;
+  animation: smoth-effet-open 0.7s;
+  top: 0;
+  color: #172b4d;
+}
+
+.modal-btn {
+  font-size: 17px;
+  text-align: center;
+  padding: 4px;
+  border-radius: 8px;
+  background: #fff;
+  border: 2px #6070ff solid;
+  outline: none;
+  color: #6070ff;
+  font-weight: 600;
+  margin-right: 5px;
+  margin-top: 0;
+}
+
+.modal-example.show {
+  display: flex;
+}
+
+.modal-example img {
+  width: 65%;
+}
+
+.modal-example .closeModal {
+  font-size: 50px;
+  color: #67798e;
+  border: none;
+  outline: none;
+  background: none;
+}
+
+.modelContainer {
+  width: 50%;
+  min-height: 40%;
+  overflow: auto;
+  padding-bottom: 100px;
+}
+
+.see-project-buttons {
+  margin-left: 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2px;
+  width: 260px;
+  margin-top: 20px;
+}
+
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  padding-top: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(193, 199, 208, 0.7);
+}
+
+.modal-content {
+  position: relative;
+  background-color: #fff;
+  margin: auto;
+  padding: 0;
+  border: 1px solid #888;
+  width: 90%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  -webkit-animation-name: animatetop;
+  -webkit-animation-duration: 0.4s;
+  animation-name: animatetop;
+  animation-duration: 0.4s;
+}
+
+@-webkit-keyframes animatetop {
+  from {
+    top: -300px;
+    opacity: 0;
+  }
+
+  to {
+    top: 0;
+    opacity: 1;
+  }
+}
+
+@keyframes animatetop {
+  from {
+    top: -300px;
+    opacity: 0;
+  }
+
+  to {
+    top: 0;
+    opacity: 1;
+  }
+}
+
+.close {
+  color: #172b4d;
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
+}
+
+.close:hover,
+.close:focus {
+  color: #172b4d;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+ul.project__lang {
+  display: flex;
+  list-style-type: none;
+  padding: 5%;
+}
+
+.project__lang a.lang1 {
+  margin: 20px 10px 20px 0;
+  background: #ebebff;
+  color: #6070ff;
+  font-weight: 500;
+  border-radius: 7px;
+  padding: 5px;
+  text-decoration: none;
+}
+
+.modal-head {
+  display: flex;
+  justify-content: space-between;
+}
+
+.short_desc {
+  display: flex;
+  align-items: center;
+}
+
+.short_desc * {
+  margin-right: 10px;
+}
+
+.short_desc h4 {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.short_desc h4:nth-child(1) {
+  color: #344563;
+  text-transform: uppercase;
+}
+
+.short_desc span {
+  background-color: #c1c7d0;
+  width: 10px;
+  height: 10px;
+  border-radius: 50%;
+}
+
+.blur {
+  filter: blur(5px);
+}
+
+@media screen and (min-width: 900px) {
+  .body {
```

Comment:
- [ ] Well done on creating the media query, but I look, your app is not responsive yet, we have some point of improvement in the mobile version as well as in the desktop one. it is asked to keep the mobile version working fine despite the changes bring for the the desktop version, please check the record below and make the necessary changes

MOBILE VERSION

- [ ] There is a presence of white space and some unnecessary padding  of the app that makes the mobile not behave fine

![5](https://github.com/gdvl777/portfolio_desktop_version_05/assets/77843050/719808b1-403b-404e-a2df-2817e06c74a5)


DESKTOP VERSION

![6](https://github.com/gdvl777/portfolio_desktop_version_05/assets/77843050/50e92dd6-9028-4993-b307-773ca516e3e3)